DirectEthernet/src/main/scala/org/labrad/ethernet/server/DirectEthernet.scala

```diff
@@ -180,6 +180,17 @@ extends ServerContext with Logging {
     }
   }
 
+  private def processPackets(packets: Seq[EthernetPacket]): Seq[(String, String, Int, Array[Byte])] = {
```

Comment:
docs are good, but in my mind the name and type signature here is really all you need to know. More text here would seem to me to just repeat what's already pretty evident in the code.